src/psyclone/tests/test_files/dynamo0p3/mixed_sci_kernel_mod.f90

```diff
@@ -0,0 +1,116 @@
+! -----------------------------------------------------------------------------
```

Comment:
It's not obvious to me what this kernel adds to what we already have in `mixed_kernel_mod.f90`. Please could you explain? (Despite its name, it still boils down to a choice between 32- and 64-bit implementations.)

Reply:
I needed a kernel to just use scalars and fields of `r_bl` and `r_phys` precisions with no operators as there are no plans to add operators with these precisions for now. I can remove the new kernel and just reuse `mixed_kernel_mod.f90` with existing operator types if you're happy with it.

Reply:
That's great, thanks.